tests/test_upload.py

```diff
@@ -41,6 +41,7 @@ def test_upload_expression(self):
         expressions_3 = self.run_processor("import:upload:expression", inputs)
         self.assertFiles(expressions_3, "rc", "00Hr_rc.tab.gz")
         self.assertFiles(expressions_3, 'exp', '00Hr_tpm.tab.gz')
+        self.assertJSON(expressions_3, expressions_3.output['exp_json'], '', 'expression_2.json.gz')
```

Comment:
@jkokosar, you should test the three different line endings. We should prepare three small expression test files.